src/components/ships/Ship.java

```diff
@@ -18,10 +18,6 @@ public abstract class Ship extends Entity{
 	
 	public Ship(String imagePath) {
 		super(imagePath);
-		this.xloc = 0.0;

```

Comment:
why is this done?

Reply:
this was already there. I can remove it

Reply:
nvm. I can put it back. I did need to delete the width and height being initialized here though because all the enemies dimensions were being set to zero rather than from the image